app/src/main/res/values-sv/strings.xml

```diff
@@ -0,0 +1,952 @@
+
+    <string name="init_youtube_missing">Överväg att installera YouTube-appen om du vill titta på trailers!</string>
+
+    <string name="text_confirm_exit">Tryck tillbaka igen för att lämna appen</string>
+
+    <string name="text_filter_applying">Använda ändringar...</string>
+
+    <string name="text_filter_restriction">Filter som inte är tillämpliga på den här sidan</string>
+
+    <string name="text_quality_changed">Innehållslista Kvalitet ändrad!</string>
+
+    <string name="text_review_changed">Ändrad granskningstyp!</string>
+
+    <string name="text_auth_unknown">Oväntad programstatus, Vänligen starta om programmet för att åtgärda det här!</string>
+
+
+    <string name="title_bottom_sheet_followers">Följare: %d</string>
+    <string name="title_bottom_sheet_following">Följer: %d</string>
+
+    <string name="text_no_likes">Inget likes att visa</string>
+
+    <string name="text_user_anime_time">Kollat på %s animes</string>
+
+    <string name="text_user_manga_chapters">Läst %s mangas</string>
+
+    <string name="text_notification">%d Underrättelse</string>
+    <string name="text_notifications">%d Anmälningar</string>
+
+    <string name="alerter_notification_title">Meddelanden tillgängliga</string>
+    <string name="alerter_notification_text">Du har en eller flera olästa anmälningar!</string>
+
+    <string name="title_new_features">Nya egenskaper!</string>
+
+    <string name="text_new_features">
+        <![CDATA[
+        Följande och följare är tillgängliga genom att knacka på etiketterna under användarbeskrivningen.
+           Du kan också klicka på Anime, Anime Time & Manga-etiketterna nedan för mer innehåll!
+        ]]>
+     </string>
+
+    <string name="text_reply_action_feedback">Taggad in i ingången!</string>
+
+    <string name="busy_please_wait">Upptagen, var god vänta!</string>
+
+    <string name="warning_empty_input">Tom inmatning är inte tillåten</string>
+
+    <string name="text_experimental_feature">Detta är en experimentell funktion!</string>
+
+    <string name="text_premium_show">Premium show. Länk ej tillgänglig!</string>
+
+    <string name="text_link_unavailable">Länk till det här avsnittet är inte tillgängligt</string>
+    <string name="error_episode_fetch">Uppfått felkod: %d</string>
+
+    <string name="waring_missing_episode_links">Nuvarande serier har inga Crunchyroll-länkar</string>
+
+    <string name="text_play_video">Spela filmen</string>
+    <string name="text_multiple_images">Flera bilder</string>
+    <string name="text_multiple_videos">Flera videor</string>
+
+    <string name="text_insufficient_genres">För lite genrer just nu!</string>
+
+    <string name="text_checking_collection">Söker i ditt bibliotek ..</string>
+
+    <string name="text_processing_request">Behandling av förfrågan ..</string>
+    <string name="text_permission_required">AniTrend beviljades inte tillstånd!</string>
+
+    <string name="text_read_more"Läs mer</string>
```

Comment:
Missing `>`
```suggestion
    <string name="text_read_more">Läs mer</string>
```

Reply:
I've also checked the rest of the file, I haven't spotted anymore issues. Hopefully everything is ok now 😃

Reply:
Thanks for having sharp eyes I think I might need glasses soon.